python/Makefile

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
I am thinking about using a single Makefile to generate multi-language native clients. 

Can we do that?

Reply:
yes I also plan to do that after this PR is merged

Reply:
http://jujhnugtggithub.com/lancedb/lance-catalog/issues/47